src/utils.js

```diff
@@ -549,6 +549,70 @@ function reduceConsecutiveBackslashes(str) {
 	return result;
 }
 
+/**
+ * Helper function to suggest a corrected format for the user provided input date
+ * @param {string} inputDate
+ */
+/*function suggestCorrectedDateFormat(inputDate) {
+	// Remove any non-numeric characters except 'T' and 'Z'
+	let correctedDate = inputDate.replace(/[^\dTZ]/g, '');
+
+	// If "T" is missing, insert it between the date and time
+	if (!/T/.test(correctedDate) && correctedDate.length >= 15) {
+		correctedDate = correctedDate.slice(0, 8) + 'T' + correctedDate.slice(8);
+	}
+
+	// Add missing characters to match the correct format
+	correctedDate = correctedDate.replace(
+		/(\d{4})(\d{2})(\d{2})T?(\d{2})(\d{2})(\d{2})Z?/,
+		'$1-$2-$3T$4:$5:$6Z',
+	);
+
+	return correctedDate;
+}*/
+
+function suggestCorrectedDateFormat(inputDate) {
```

Comment:
What are the primary corrections we're providing? Basically adding missing non-numeric characters like `T`, `-`, and `:`?